packages/code-du-travail-nlp/Dockerfile

```diff
@@ -10,7 +10,6 @@ RUN python -m venv venv
 RUN . venv/bin/activate
 RUN pip install -r requirements.txt
 
-RUN mkdir -p ./data
```

Comment:
ah I see data was created but not copied to the host filesystem :(
```diff
+ COPY ./data ./data 
```
would fix the problem to

Reply:
ahhh, ok, nice catch !

Reply:
Humm non ça ne fonctionne pas :/ Si data existe pas, le mkdir est peut-être fait sur l'image mais pas en local ?